packages/apps/composer-app/src/App.tsx

```diff
@@ -44,7 +44,7 @@ export const App = () => {
   });
 
   return (
-    <UiKitProvider
+    <UiProvider
```

Comment:
Will eventually come from appkit?

Reply:
No, `react-components` needs to provide the provider. We can rename it as needed, but it's necessary for several of the components it offers.